core/model/modx/modparser.class.php

```diff
@@ -135,6 +135,12 @@ public function collectElementTags($origContent, array &$matches, $prefix= '[[',
             if (($stopPos= strrpos($origContent, $suffix)) === false) {
                 return $matchCount;
             }
+            $uniqueCharacters = count_chars($prefix, 3);
```

Comment:
For future people coming across this block of code wondering what it does (like me, right now, with a fried brain), could you add some descriptive comments? I'm struggling to understand the way this intends to work for the issue it solves.

Reply:
Good point! I will document the changes a bit more later.

Reply:
Thanks!

Reply:
@Mark-H done!